dev/run-tests.py

```diff
@@ -227,11 +228,32 @@ def build_spark_documentation():
     os.chdir(SPARK_HOME)
 
 
+def get_zinc_port():
+    """
+    Get a randomized port on which to start Zinc
+    """
+    return random.randrange(3030, 4030)
```

Comment:
I hope this doesn't lead to builds failing because zinc can't bind to the chosen port.